cmd/podman/pod_kill.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"syscall"
+
+	"fmt"
+	"github.com/docker/docker/pkg/signal"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	podKillFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "Kill all containers in all pods",
+		},
+		cli.StringFlag{
+			Name:  "signal, s",
+			Usage: "Signal to send to the pod",
```

Comment:
If you mention containers above, you should mention them here.

Signall all containers in the pod.